### PR TITLE
AUI-3084 Reset the height monitor so that the text entered in the text area does not persist across pages

### PR DESCRIPTION
**Pull request by jonathanmccann**
Hey Jon,

I haven't submitted an AUI fix for a while so let me know if I missed something or if this is the incorrect branch.

The issue is that text inputted in the System Settings on DXP is placed into a "pre" element in order to calculate the height the text area should be. This causes a problem because these "pre" elements are not cleared out and appear on other pages. If you think there's a better way to resolve this issue, please feel free to let me know.

Thanks!

### CODE REVIEW

**Comment by jonmak08** [comment]
Hey @jonathanmccann,

I do not think this is correct fix since you are basically clearing out the contents of the hidden field.  What I found was that when editing and saving the Control Panel -> Configuration -> System Settings -> Categories Navigation, a ```<pre />``` element gets added to the ```<body>``` of the page.  The only reason it is not visible is because of the CSS class ```helper-hidden-accessible``` hides it.

When you go to Control Panel -> Configuration -> Server Administration -> Data Migration and click the ```Execute``` button, you get redirected to a page where the ```<pre />``` elements with the CSS class ```helper-hidden-accessible``` are still on the page, but because the deprecated AUI CSS classes are not brought in on the page, the code is still visible.

I think the right fix would either to bring the CSS styles in on the "Shut down message" page or to investigate why the code in the ```<pre />``` tags are added to the page to begin.

Can you please review my comments and look into this issue further and let me know your findings?  Thanks!

**Comment by holatuwol** [comment]
@jonmak08 The heightMonitor is the hidden `<pre />` tag, not a hidden field.

https://github.com/jonathanmccann/alloy-ui/blob/9d4f5bdb41ccd2011b890e93414453ef041a58c9/src/aui-autosize-deprecated/js/aui-autosize-deprecated.js#L85

The `<pre />` tags were added to the page in order to compute a proper height for the text area field, and so we believe it no longer needs that once that height has been calculated (nothing ever looks at the height monitor after this).

https://github.com/jonathanmccann/alloy-ui/blob/9d4f5bdb41ccd2011b890e93414453ef041a58c9/src/aui-autosize-deprecated/js/aui-autosize-deprecated.js#L186-L202

In summary, the code change doesn't clear out the content of a hidden input field, but rather the content of the hidden `<pre />` tag, so clearing out the hidden `<pre /`> tag when its contents are no longer needed appears to be the right fix. Please let us know if we're missing something.

**Comment by jonmak08** [comment]
Hey @holatuwol,

Gotcha.  I did not realize that it was creating and reusing it's own node.  I thought it was looking up and reusing the node on the page made elsewhere.

I will reopen and review.

Please let me know if you have any questions. Thanks!

**Comment by jonmak08** [comment]
Merged and pushed.  Thanks!
